composer/trainer/trainer.py

```diff
@@ -612,7 +612,7 @@ class Trainer:
             .. note:: This is implemented by taking the batch yielded by the ``train_dataloader`` and splitting
                 it into ``grad_accum`` sections. Each section is of size ``train_dataloader // grad_accum``.
                 If the batch size of the dataloader is not divisible by ``grad_accum``,
-                then the last section will be of size ``batch_size % grad_accum``.
+                then the last section will be of size ``batch_size mod grad_accum``.
```

Comment:
I don't think this is more correct. `mod` isn't really an operator in its mathematical sense. `%` is, and programmers know what it means.

Reply:
Can we just escape the `%` instead?

Reply:
It will then also escape in the docs build. I could do `%%` for now but don't want to have to push a new yahp version.

Reply:
Discussed offline; will leave as-is for now. Created https://mosaicml.atlassian.net/browse/CO-558 to track it.